gradle/publish.gradle

```diff
@@ -13,14 +14,15 @@ task testJar(type: Jar) {
 
 publishing {
     publications {
-        bintray(MavenPublication) {
+        artifactory(MavenPublication) {
             from components.java
             artifact(sourceJar) {
                 classifier 'sources'
             }
             artifact(testJar) {
                 classifier 'tests'
             }
+            version project.version.replaceAll(/-\d+.*/, '-SNAPSHOT')
```

Comment:
Why `.*`? I would think that it would get rid of `-dirty` but we should never have that on circle.